xrootd/client.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package xrootd // import "go-hep.org/x/hep/xrootd"
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"os"
+
+	"go-hep.org/x/hep/xrootd/encoder"
+	"go-hep.org/x/hep/xrootd/streammanager"
+)
+
+var logger = log.New(os.Stderr, "xrootd: ", log.LstdFlags)
+
+// A Client to xrootd server
```

Comment:
`Client` is the main type people will have to deal with.
It could deserve a bit more of documentation :)

Reply:
I have added a bit more of documentation, but I'm unsure if that's enough and what else I can add.